src/groupby.jl

```diff
@@ -447,11 +447,41 @@ end
 
 Generate a `Vector` of `UnitRange` with length `step(A)`
 """
-intervals(rng::AbstractRange) = IntervalSets.Interval{:closed,:open}.(rng, rng .+ step(rng))
+intervals(rng::AbstractRange) =
+    IntervalSets.Interval{:closed,:open}.(rng, rng .+ step(rng))
 
 """
     ranges(A::AbstractRange{<:Integer})
 
 Generate a `Vector` of `UnitRange` with length `step(A)`
 """
 ranges(rng::AbstractRange{<:Integer}) = map(x -> x:x+step(rng)-1, rng)
+
+
+"""
+    combine(f::Function, gb::DimGroupByArray; dims=:)
+
+Combine the `DimGroupByArray` using funciton `f` over the group dimensions.
+
+If `dims` is given, combine only the dimensions in `dims`. The reducing function
+`f` must accept a `dims` keyword.
+"""
+function combine(f::Function, gb::DimGroupByArray{G}; dims=:) where G
```

Comment:
DataAPI.combine?  Since DD owns one of the types...

Reply:
It doesn't exist yer https://github.com/JuliaData/DataAPI.jl/issues/65 ;)